blocks/carousel/carousel.css

```diff
@@ -0,0 +1,161 @@
+/* Carousel block styles */
+.carousel {
+    position: relative;
+    width: 100%;
+    height: 100%;
+    order: 1;
+    display: flex;
+    flex-flow: row wrap;
+    align-content: flex-start;
+    align-items: flex-start;
+    justify-content: center;
+    margin: 0 auto;
+    text-align: center;
+}
+
+.carousel .carousel-slide-container {
+    width: 100%;
+    max-width: 744px;
+    position: relative;
+    overflow: hidden;
+    margin: 0 auto;
+    transition: height 0.2s linear
+}
+
+.carousel .carousel-slide {
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    transition: all 0.5s;
+}
+
+.carousel .carousel-slide > div:nth-child(1) {
+    width: 100%;
+    font-size: var(--font-size-32);
+    line-height: 120%;
+    letter-spacing: .0175em;
+    font-weight: var(--font-weight-regular);
+    margin: 0 auto;
+    text-align: center;
+}
+
+.carousel .carousel-slide figure {
+    align-content: center;
+    align-items: center;
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: space-between;
+    margin: 32px auto 0;
+    width: fit-content;
+}
+
+.carousel .carousel-slide figure img {
+    border-radius: 50%;
+    height: 64px;
+    width: 64px;
+    margin-bottom: 0;
+    margin-right: 24px;
+}
+
+.carousel .carousel-slide figcaption.caption {
+    font-family: var(--sans-serif-font-semibold);
+    font-size: var(--font-size-16);
+    letter-spacing: var(--letter-spacing-001-em);
+    line-height: var(--line-height-160);
+    font-weight: var(--font-weight-medium);
+}
+
+.carousel .carousel-nav {
+    opacity: 0;
+    z-index: 2;
+    top: 125px;
+    height: 54px;
+    width: 54px;
+    position: absolute;
+    transition: .5s;
+    cursor: pointer;
+    background: url('../../icons/testimonial.svg');
```

Comment:
We have the standard purple arrow already in the icons folder. You could reuse the same. There's also a gradient in the global vars you can use for hover effect for the colorful hover state